calibrate.py

```diff
@@ -5,27 +5,43 @@
 import fileinput
 import SPEFile
 import os
+
 energy_list = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
                2614.51]
 cal_headers = [351.93, 583.19, 609.31, 911.20, 1460.82, 1764.49,
                2614.51, 'Output']
 
-
 def acquire_files():
     """
     acquire_files gathers all the .Spe file in the current file directory and
     returns a list containing all .Spe files.
-    Files is a list of .spe files.
     """
     sample_measurements = []
+    sample_names = []
+    f = []
     dir_path = os.getcwd()
     for file in os.listdir(dir_path):
         if file.lower().endswith(".spe"):
+            # Ignore the background and reference spectra
             if file == "USS_Independence_Background.Spe":
                 pass
+            elif file == "UCB018_Soil_Sample010_2.Spe":
+                pass
             else:
+                if '_recal' in file:
+                    f.append(file)
                 sample_measurements.append(file)
-    return sample_measurements
+
+    sample_measurements.sort()
+
+    for i in np.arange(len(f)):
+        sample_measurements.remove(f[i].replace('_recal', ''))
```

Comment:
The code should handle the situation where there is e.g. `spectrum_recal.spe` but no `spectrum.spe`. Right now it would give an error on this line because it's trying to `remove` an element that doesn't exist.